db/migrations/20230102163643_create_payments_table.sql

```diff
@@ -0,0 +1,13 @@
+-- migrate:up
+CREATE TABLE payments (
+    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    payType VARCHAR(100) NOT NULL,
+    amount INT NOT NULL,
+    orderId VARCHAR(100) NOT NULL,
+    orderName VARCHAR(200) NOT NULL,
+    requestedAT VARCHAR(200) NOT NULL,
+    approvedAT VARTCHAR(200) NOT NULL
```

Comment:
이전 테이블들의 컬럼은 모두 snake_case로 작성되어 있습니다.
payments 테이블의 컬럼도 snake_case로 수정해주세요!

- 토스 공식문서에 보면 `카드` 혹은 `간편결제` 와 같은 결제 방식은 method라는 key 값을 사용하고 있습니다.
- amount는 책의 가격을 가리킵니다. 현재 서비스에서 책 가격은 decimal 필드로 관리하고 있습니다.
- `requested_at` 과 `approved_at`컬럼은 실제 결제를 요청하고 결제가 승인된 시간을 가리키는 데이터입니다. 문자열보다는 날짜를 가리키는 DATETIME 또는 TIMESTAMP 로 관리하는 것이 좋을 것 같습니다.
- 결제를 진행하다가 취소할 경우도 있으니 status 컬럼을 통해 관리해주세요!

```suggestion
    method VARCHAR(100) NOT NULL,
    amount DECIMAL(10, 2) NOT NULL,
    order_id VARCHAR(100) NOT NULL,
    order_name VARCHAR(200) NOT NULL,
    status VARCHAR(200) NULL,
    requested_at DATETIME NULL,
    approved_at DATETIME NULL
```